e2e/tests/nightly/admin/workspace-admin-ui.spec.ts

```diff
@@ -0,0 +1,150 @@
+import { signInWithAccessToken } from 'utils/test-utils';
+import { config } from 'resources/workbench-config';
+import navigation, { NavLink } from 'app/component/navigation';
+import WorkspaceAdminPage from 'app/page/admin-workspace-page';
+import { ConceptSetSelectValue, CloudStorageHeader, Language } from 'app/text-labels';
+import RuntimePanel from 'app/component/runtime-panel';
+import WorkspacesPage from 'app/page/workspaces-page';
+import WorkspaceCard from 'app/component/workspace-card';
+import WorkspaceDataPage from 'app/page/workspace-data-page';
+import AdminNotebookPreviewPage from 'app/page/admin-notebook-preview-page';
+import { waitForDocumentTitle } from 'utils/waits-utils';
+import { Page } from 'puppeteer';
+import NotebookPreviewPage from 'app/page/notebook-preview-page';
+
+describe('Workspace Admin', () => {
+  const workspaceName = 'e2eAdminWorkspace';
+  const pyNotebookName = 'e2eAdminNotebook';
+  let workspaceNamespace = '';
+
+  beforeEach(async () => {
+    await signInWithAccessToken(page, config.ADMIN_TEST_USER);
+  });
+
+  test('Verify the lock-Workspace feature and the Workspace Admin page UI', async () => {
+    const workspacesPage = new WorkspacesPage(page);
+    await workspacesPage.load();
+    await workspacesPage.createWorkspace(workspaceName);
+    await waitForDocumentTitle(page, 'Data Page');
+    const dataPage = new WorkspaceDataPage(page);
```

Comment:
Missing `await dataPage.waitForLoad`. Please check other places where this call is also needed.